lib/elixir/lib/process.ex

```diff
@@ -784,6 +837,55 @@ defmodule Process do
   @spec hibernate(module, atom, list) :: no_return
   defdelegate hibernate(mod, fun_name, args), to: :erlang
 
+  @type alias_opt :: :explicit_unalias | :reply
+
+  @typedoc """
+  An alias returned by `alias/0` or `alias/1`.
+
+  See [the module documentation](#module-aliases) for more information about aliases.
+  """
+  @type alias :: reference
+
+  @doc """
+  Creates a *process alias*.
+
+  This is the same as calling `alias/1` as `alias([:explicit_unalias])`. See
+  also `:erlang.alias/0`.
+  """
+  @doc since: "1.15.0"
+  @spec alias() :: alias
+  defdelegate alias(), to: :erlang
+
+  @doc """
+  Creates a *process alias*.
+
+  See [the module documentation](#module-aliases) for more information about aliases.
+  See also `:erlang.alias/1`.
+  """
+  @doc since: "1.15.0"
+  @spec alias([alias_opt]) :: alias
+  defdelegate alias(options), to: :erlang
+
+  @doc """
+  Explicitly deactivates a process alias.
+
+  Returns `true` if `alias` was a currently-active alias for current processes,
+  or `false` otherwise.
+
+  See [the module documentation](#module-aliases) for more information about aliases.
+  See also `:erlang.unalias/1`.
+
+  ## Examples
+
+      alias = Process.alias()
+      Process.unalias(alias)
+      #=> true
+
+  """
+  @doc since: "1.15.0"
+  @spec unalias(alias) :: boolean
+  defdelegate unalias(alias), to: :erlang
```

Comment:
Should we go ahead and inline those functions in the compiler too?

We should also support the alias option on `Process.monitor`. :)

Reply:
Yes, we should inline those. I was planning to support this in `Process.monitor/2` in another PR but no problem, I'll make those changes too 🙃

Reply:
As you prefer!